src/engines/misc/external/audio.hpp

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include "core/audio/audio_buffer_pool.hpp"
+#include "engines/misc/sends/audio.hpp"
+#include "external.hpp"
+#include "services/log_manager.hpp"
+
+namespace otto::engines::external {
+
+  using namespace core;
+
+  struct Audio {
```

Comment:
move implementations of all of these methods to `audio.cpp`

Reply:
Done